# offset, d, a, alpha
dh_matrix: [ 0.0, 0.0885,   0.0, rad(-pi/2),
             0.0,    0.0, -0.10,        0.0,
             0.0,    0.0, 0.138,  rad(pi/2)]
end_effector:
  x: 0.0
  y: 0.0
  z: 0.04961
  roll: rad(-pi/2)
  pitch: 0.0
  yaw: rad(-pi/2)